docs/user-guide/cli-using-editing-team-configuration.md

```diff
@@ -0,0 +1,43 @@
+# Editing team configurations
+
+After you [initialize team configuration](../user-guide/cli-using-initializing-team-configuration), the newly created team profiles need additional details in order to be applied in your environment. This could include information such as a port number, or user credentials.
+
+You might also need to modify the configuration file to add new profiles for accessing mainframe services.
+
+## Adding, modifying team profiles
+
+To define additional mainframe services and other profiles in an existing team configuration file:
```

Comment:
For these instructions, it might be helpful to note that we are modifying "an existing **global** team configuration file"